RobotFramework_DoIP/DoipKeywords.py

```diff
@@ -0,0 +1,92 @@
+from robot.api.deco import keyword
+from robot.api import logger
+from robot.libraries.BuiltIn import BuiltIn
+
+from doipclient import DoIPClient
+
+class DoipKeywords(object):
+
+    def __init__(self):
+        address, announcement = DoIPClient.get_entity()
+        logical_address = announcement.logical_address
+        ip, port = address
+        print(ip, port, logical_address)
+
+        self.ip = ip
+        self.logical_address = logical_address
+        logger.info(f"IP: {ip} ")
+
+    @keyword
+    def connect(self):
```

Comment:
Hi Thong,
as per https://python-doipclient.readthedocs.io/en/latest/
DoIPClient has the interface:
(ecu_ip_address, ecu_logical_address, tcp_port=13400, udp_port=13400, activation_type=ActivationType.Default, protocol_version=2, client_logical_address=3584, client_ip_address=None, use_secure=False, auto_reconnect_tcp=False)

This interface need to be available as keyword if the user don't want to use get_entity, too for new connection.

Thank you,
 Thomas

Reply:
Hi Thomas, 

I have updated it.

Thank you,
Thong Hua